tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ProcessStart/EnvironmentVariablesAnalyzer.cs

```diff
@@ -0,0 +1,52 @@
+// <copyright file="EnvironmentVariablesAnalyzer.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Process
+{
+    internal static class EnvironmentVariablesAnalyzer
```

Comment:
I suggest renaming this to `EnvironmentVariablesScrubber` to make it more clear what it does.

Reply:
Done!